### ISSUE TITLE
[checkpoints/deepcrack/opt.txt]  DeepCrack model parameters issue

### ISSUE THREAD

**Issue by WenkeC** [yhlleo/DeepSegmentor#6]
Thank you for your work and code. I run the code according to the method you provided, but my result is much lower than yours. 
There are some differences between the parameters in the train_deepcrack.sh file and those in your paper, such as learning rate decay. Maybe this is what caused the lower results when I train the Deepcrack-BN model.
So could you provide the opt.txt file of your Deepcrack-BN and Deepcrack-Aug models? I am sorry to trouble you, thank you again.

**Comment by yhlleo** [comment]
Thanks for your concern. I also find that the Pytorch implementation achieves lower performances than the results reported in the paper by using Caffe framework (I tried with the same type loss and data).

**Comment by yhlleo** [comment]
@WenkeC , I updated the codes for improving the performances of PyTorch implementation. You can find the notice [here](https://github.com/yhlleo/DeepSegmentor/blob/master/eval/README.md).

**Comment by WenkeC** [comment]
Thanks for your update, I will retrain the model.
